src/commands/protocols/update/receiver/submit-update-commit-command.js

```diff
@@ -72,6 +72,12 @@ class SubmitUpdateCommitCommand extends EpochCommand {
             epoch,
             async (result) => {
                 if (!result.error) {
+                    this.operationIdService.emitChangeEvent(
```

Comment:
why do we do `that = this` before the callback in `submitCommitCommand`, but not here ?

Reply:
We also should add it here. 
And why we set that because this frequently changes when you change the scope by calling a new function, you can't access the original value by using it. Aliasing it to that allows you still to access the original value of this.

Reply:
there is no need for that in es6 arrow functions : https://hacks.mozilla.org/2015/06/es6-in-depth-arrow-functions/